e2e/utils.ts

```diff
@@ -253,6 +287,9 @@ export const prepareStandaloneSetup = (fixtureName: string) => {
           writeFileSync(f, JSON.stringify(pkg, null, 2), 'utf8');
         }
       }
+      if (packageManager !== 'pnpm') {
+        patchMonorepoPackageJson(standaloneDir);
+      }
```

Comment:
Let me know if there are some better ways. I mean, I'm not happy with this.
Feel free to send a PR.